client/landing/jetpack-cloud/sections/scan/history/index.jsx

```diff
@@ -1,17 +1,159 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
 /**
  * External dependencies
  */
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
+import FoldableCard from 'components/foldable-card';
+import FormattedHeader from 'components/formatted-header';
+import SimplifiedSegmentedControl from 'components/segmented-control/simplified';
 import { getSelectedSiteId } from 'state/ui/selectors';
 
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const filterOptions = [
+	{ value: 'all', label: 'All' },
+	{ value: 'fixed', label: 'Fixed' },
+	{ value: 'ignored', label: 'Ignored' },
+];
+
+const scanEntries = [
+	{
+		id: 1,
+		title: 'Infected core file: index.php',
+		action: 'fixed',
+		detectionDate: '23 September, 2019',
+		actionDate: '1 October, 2019',
+		details: {
+			title: 'Unexpected string was found in: /htdocs/wp-admin/index.php',
+			description: '',
+		},
+	},
+	{
+		id: 2,
+		title: 'Infected Plugin: Contact Form 7',
+		action: 'ignored',
+		detectionDate: '17 September, 2019',
+		actionDate: '24 September, 2019',
+		details: {
+			title:
+				'Unexpected file baaaaaad--file.php contains malicious code and is not part of the Plugin',
+			description: '',
+		},
+	},
+	{
+		id: 3,
+		title: 'Infected Plugin: Contact Form 7',
+		action: 'ignored',
+		detectionDate: '16 September, 2019',
+		actionDate: '24 September, 2019',
+		details: {
+			title:
+				'Unexpected file veryBadFile.php contains malicious code and is not part of the Plugin',
+			description: '',
+		},
+	},
+	{
+		title: 'Infected core file: index.php',
+		action: 'fixed',
+		detectionDate: '16 September, 2019',
+		actionDate: '24 September, 2019',
+		details: {
+			title: 'Unexpected string was found in: /htdocs/wp-admin/index.php',
+			description: '',
+		},
+	},
+	{
+		id: 4,
+		title: 'Infected Plugin: Classic Editor',
+		action: 'ignored',
+		detectionDate: '14 September, 2019',
+		actionDate: '14 September, 2019',
+		details: {
+			title:
+				'Unexpected file veryBadFile.php contains malicious code and is not part of the Plugin',
+			description: '',
+		},
+	},
+];
+
+class ScanHistoryEntry extends Component {
+	static propTypes = {
+		entry: PropTypes.object,
+	};
+
+	renderEntryHeader() {
+		const { entry } = this.props;
+
+		const actionClass = 'entry' + entry.action === 'fixed' ? ' is-fixed' : ' is-ignored';
+
+		return (
+			<div className={ actionClass }>
```

Comment:
Can we use the <LogItem component that is found in `landing/jetpack-cloud/components` instead of creating our own?

Reply:
Sure! The only thing that makes me uncomfortable is that this component has a `highlight` property to set its color. The options are 'info', 'success', 'warning', and 'error', and neither of those matches the color we need (--studio-jetpack-green-40). Should we use it anyway and override the color from the parent component? Is that what we do in a case like this?

Reply:
I faced the same question when working on the badge (fixed or ignored just before the title of the entry). There is Badge component, is looks similar to what is in the designs. It also accepts the same `highlight` property. Should I use that o create a new one?

Reply:
I wrote that component when the colors weren't finalized. Feel free to adjust them!